### ISSUE TITLE
Optional: Allow item comparison to require identical lore/name and disallow items with more lore

### ISSUE THREAD

**Issue by Blackixx** [Blackixx/BossShopPro#26]
Issue by BSP user:

"On my server we use custom lore and name text to create items of value for players. What I am trying to do is make a shop that allows the selling of Diamond, Diamond(named ruby), diamond(named jade) ect. When I got to sell Rubies, only ruby named items sell. After setting this up I can sell Rubies by clicking the Ruby Named item, and only rubies sell. However when I sell normal diamonds, Ruby, Jade and normal diamonds are sold.

If I set the name to Diamond in the itemshopconfig as a requirement for the default diamond (Like you do with requirement of colored names) the Plugin sees that as a Diamond renamed to Diamond and not the normal dropped diamonds. Is there a way to get it to only buy the default diamonds?"

So basically new config entry: If enabled, players need to have identical items when selling to shop. Items with additional lore can longer be sold and items with a name, although the shop item has no custom name, can also no longer be sold by players if the option is enabled.
